.travis.yml

```diff
@@ -50,7 +50,7 @@ script:
       travis_terminate 1
       ;;
     esac'
-  - pip install -r requirements.txt
```

Comment:
Absolutely not, welcome to hell :D 

Maybe the way to go is to have different `requirements.txt` files, one for each version? I don't know :/
But I'd like to have the same setup process for users and the CI.